Example/EpoxyExample/ViewControllers/LayoutGroups/EntirelyInlineViewController.swift

```diff
@@ -0,0 +1,88 @@
+// Created by Tyler Hedrick on 3/24/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import EpoxyCollectionView
+import EpoxyLayoutGroups
+import Foundation
+import UIKit
+
+/// This view controller shows how you can create entire components
+/// inline using VGroupView and HGroupView inside of Epoxy
+class EntirelyInlineViewController: CollectionViewController {
+
+  // MARK: Lifecycle
+
+  init() {
+    super.init(layout: UICollectionViewCompositionalLayout.list)
+    setItems(items, animated: false)
+  }
+
+  // MARK: Internal
+
+  override var title: String? {
+    get { "Entirely Inline" }
+    set { }
+  }
+
+  var items: [ItemModeling] {
```

Comment:
nit: using an @ItemModelBuilder will reduce a level of indentation here as you can remove the `[…]`
```suggestion
  @ItemModelBuilder
  var items: [ItemModeling] {
```